apps/etterlatte-behandling/src/main/kotlin/config/ConfigUtils.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.etterlatte.config
+
+import com.typesafe.config.Config
+
+fun samle(config: Config, env: Map<String, String>): Map<String, String> {
```

Comment:
Her burde vi kun bruke en delene.

Reply:
Liker tanken på å forholde seg til en datastruktur. Høres veldig rart ut at ktor ikke støtter oss med dette ut av boksen? Finnes det ingen tilsvarende måte som feks spring gjør?

Reply:
Nja, Unleash-konfigen vår treng i grunn berre applikasjonsnamnet, clusternamnet, URL og ein enabled-brytar (for testane).

_Men_ både applikasjonsnamn og clusternamn ligg allereie definert i env, så det burde halde at vi aktiverer unleash (den trur eg for så vidt vi kunne defaulta til true, og så overstyrt ved behov i testane, @chrissearle) pluss URL i konfig. Kunne vorte ein oneliner med dette her (eller tilsvarande) heller enn å måtte setje alle fire i kvar application.conf, som er både dupliserande og (ikkje minst) lett å rote til/gløyme.